vhdx/src/vhdx_metadata.rs

```diff
@@ -303,7 +303,7 @@ impl MetadataTableEntry {
     fn new(buffer: &[u8]) -> Result<MetadataTableEntry> {
         let mut metadata_table_entry = unsafe { *(buffer.as_ptr() as *mut MetadataTableEntry) };
 
-        let uuid = crate::uuid_from_guid(buffer).map_err(VhdxMetadataError::InvalidUuid)?;
```

Comment:
You should delete this error from the enum.

Reply:
I did but then I realized it was used somewhere else.